tools/enso4igv/src/main/resources/org/enso/tools/enso4igv/enso.tmLanguage.json

```diff
@@ -85,6 +85,16 @@
 						"matches": "\\\""
 					}]
 				},
+				{
+					"name": "string.quoted.tripple.begin",
```

Comment:
`tripple` -> `triple`?

Reply:
Good point. At least that is the way it is spelled in the specs at https://macromates.com/manual/en/language_grammars

Reply:
Fixed in [ff9fcd6](https://github.com/enso-org/enso/pull/7539/commits/ff9fcd6fbb4dd608bc075ececfffd1392eaeb006) - thank you.